### PR TITLE
feat!: replace skeleton-carousel with cosmoz-slider

### PR DESCRIPTION
**Pull request by cristinecula**
BREAKING CHANGE: dropped show-dots and currentImage

### CODE REVIEW

File: cosmoz-image-viewer.html.js [hunk 1]
```diff
@@ -142,9 +105,15 @@ export const template = html`
...
+
+	cosmoz-slider { 
+		min-height: 150px;
+		overflow-y: auto !important;
+		height:100%;
+	}
 </style>
 
-<div>
+<div style="height: 100%">
```
Comment (megheaiulian) [inline cosmoz-image-viewer.html.js:116]: Do we need this wrapper or can we remove it? :host is enough wrapper ?

**Review by megheaiulian — APPROVED** [review]
:+1:

**Comment by megheaiulian** [comment]
Code looks very good but tests are timing out ...

File: cosmoz-image-viewer.html.js (hunk 1 above)
Comment (cristinecula) [reply cosmoz-image-viewer.html.js:116]: I'll try to remove it.

Comment (cristinecula) [reply cosmoz-image-viewer.html.js]: @megheaiulian I've removed it and it works fine.

**Pull request merged**